backend/tests/core/test_config.py

```
@@ -78,6 +78,22 @@ def test_settings_from_env_variables(self, monkeypatch):
+    ):
+        env_file = tmp_path / ".env"
+        env_file.write_text(
+            "GIT_TOKEN_AES_KEY=dotenv-key\nGIT_TOKEN_AES_IV=dotenv-iv\n",
+            encoding="utf-8",
+        )
+        monkeypatch.delenv("GIT_TOKEN_AES_KEY", raising=False)
+        monkeypatch.setenv("GIT_TOKEN_AES_IV", "process-iv")
+
+        load_git_token_crypto_environment(env_file)
+
+        assert os.environ["GIT_TOKEN_AES_KEY"] == "dotenv-key"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "GIT_TOKEN_AES_KEY"

(S105)